modules/users/server/controllers/user-auth.controller.js

```diff
@@ -466,7 +466,7 @@ function userAuthController(logger, shared) {
    * @return {Promise}
    */
   function sendPasswordChangeEmail(user) {
-    const url = 'http://' + config.app.host + ':' + config.app.port_http + '/reset-password;token=' + user.resetPassword.token;
+    const url = authHelpers.generateUrl() + '/changePassword;token=' + user.verification.token;
```

Comment:
Is changePassword the right URI?  I forget what it was, but I wouldn't want that to break...